compiler/test/stdlib/number.test.gr

```diff
@@ -444,13 +444,19 @@ assert Number.sin(0.0) == 0
 assert Number.sin(1) > 0.84147 && Number.sin(1) < 0.84148
 assert Number.sin(-1) < -0.84147 && Number.sin(-1) > -0.84148
 assert Number.sin(20) > 0.91294 && Number.sin(20) < 0.91295
+assert Number.sin(1/2) > 0.4794 && Number.sin(1/2) < 0.4795
+// TODO(#1478): Correct this test
```

Comment:
Could you update these to say "Update this test when sin is unbounded"?

Reply:
```suggestion
// TODO(#1478): Update this test when sin is unbounded
```